tests/pandas/test_continuous.py

```diff
@@ -0,0 +1,122 @@
+"""
+Contains unit tests for scores.continuous.standard
+"""
+# pylint: disable=missing-function-docstring
+# pylint: disable=line-too-long
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import scores.pandas as scores
+
+PRECISION = 4
+
+# Mean Squared Error
+
+
+def test_mse_pandas_series():
+    """
+    Test calculation works correctly on pandas series
+    """
+
+    fcst_pd_series = pd.Series([1, 3, 1, 3, 2, 2, 2, 1, 1, 2, 3])
```

Comment:
You should add a NaN to the fcst and obs inputs in all of your tests. Just combine it in the one test rather than having multiple like you do with RMSE

Reply:
These tests were copied from `tests/continuous/test_standard.py` to show no breaking changes had occured. I am happy to improve these tests however